commit e834e1e934ed0af673598d8c0c34afb2af56bee0
Author: Jia Tan <jiat0218@gmail.com>
Date:   Fri Jan 6 20:35:55 2023 +0800

    Build: Add missing stream_decoder_mt.c to .vcxproj files.
    
    The line in the .vcxproj files for building with was missing in 5.4.0.
    Thank to Hajin Jang for reporting the issue.

diff --git a/windows/vs2013/liblzma.vcxproj b/windows/vs2013/liblzma.vcxproj
index 5ec6f627..2c8e399b 100644
--- a/windows/vs2013/liblzma.vcxproj
+++ b/windows/vs2013/liblzma.vcxproj
@@ -248,6 +248,7 @@
     <ClCompile Include="..\..\src\liblzma\common\stream_buffer_decoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_buffer_encoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_decoder.c" />
+    <ClCompile Include="..\..\src\liblzma\common\stream_decoder_mt.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_encoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_encoder_mt.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_flags_common.c" />
diff --git a/windows/vs2013/liblzma_dll.vcxproj b/windows/vs2013/liblzma_dll.vcxproj
index c6554e2b..4dede9f0 100644
--- a/windows/vs2013/liblzma_dll.vcxproj
+++ b/windows/vs2013/liblzma_dll.vcxproj
@@ -280,6 +280,7 @@
     <ClCompile Include="..\..\src\liblzma\common\stream_buffer_decoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_buffer_encoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_decoder.c" />
+    <ClCompile Include="..\..\src\liblzma\common\stream_decoder_mt.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_encoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_encoder_mt.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_flags_common.c" />
diff --git a/windows/vs2017/liblzma.vcxproj b/windows/vs2017/liblzma.vcxproj
index 9bf5cc38..f00fd21f 100644
--- a/windows/vs2017/liblzma.vcxproj
+++ b/windows/vs2017/liblzma.vcxproj
@@ -248,6 +248,7 @@
     <ClCompile Include="..\..\src\liblzma\common\stream_buffer_decoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_buffer_encoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_decoder.c" />
+    <ClCompile Include="..\..\src\liblzma\common\stream_decoder_mt.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_encoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_encoder_mt.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_flags_common.c" />
diff --git a/windows/vs2017/liblzma_dll.vcxproj b/windows/vs2017/liblzma_dll.vcxproj
index 7f9a18fb..66f4279b 100644
--- a/windows/vs2017/liblzma_dll.vcxproj
+++ b/windows/vs2017/liblzma_dll.vcxproj
@@ -280,6 +280,7 @@
     <ClCompile Include="..\..\src\liblzma\common\stream_buffer_decoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_buffer_encoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_decoder.c" />
+    <ClCompile Include="..\..\src\liblzma\common\stream_decoder_mt.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_encoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_encoder_mt.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_flags_common.c" />
diff --git a/windows/vs2019/liblzma.vcxproj b/windows/vs2019/liblzma.vcxproj
index c373cd3e..cc021470 100644
--- a/windows/vs2019/liblzma.vcxproj
+++ b/windows/vs2019/liblzma.vcxproj
@@ -249,6 +249,7 @@
     <ClCompile Include="..\..\src\liblzma\common\stream_buffer_decoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_buffer_encoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_decoder.c" />
+    <ClCompile Include="..\..\src\liblzma\common\stream_decoder_mt.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_encoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_encoder_mt.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_flags_common.c" />
diff --git a/windows/vs2019/liblzma_dll.vcxproj b/windows/vs2019/liblzma_dll.vcxproj
index b30bf3ed..7c3a351a 100644
--- a/windows/vs2019/liblzma_dll.vcxproj
+++ b/windows/vs2019/liblzma_dll.vcxproj
@@ -281,6 +281,7 @@
     <ClCompile Include="..\..\src\liblzma\common\stream_buffer_decoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_buffer_encoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_decoder.c" />
+    <ClCompile Include="..\..\src\liblzma\common\stream_decoder_mt.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_encoder.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_encoder_mt.c" />
     <ClCompile Include="..\..\src\liblzma\common\stream_flags_common.c" />